packages/codd/hooks/codd-validate-precommit.py

```
@@ -114,22 +437,586 @@ def _codd_config_path(root: str) -> Path:
+        target_real = target.resolve(strict=False)
+    except OSError:
+        return None
+    try:
+        target_real.relative_to(base_real)
+    except ValueError:
+        return None
+    return target_real
+
+
+def _copy_no_follow(src: Path, dest: Path) -> bool:
+    """`dest` が symlink でも辿らず、内容を `src` で安全に置き換える（A-1: Issue #338 反復4）。
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)